connector/connect/src/main/protobuf/spark/connect/expressions.proto

```diff
@@ -83,59 +80,39 @@ message Expression {
     // directly declare the type variation).
     uint32 type_variation_reference = 51;
 
-    message VarChar {
-      string value = 1;
-      uint32 length = 2;
-    }
-
     message Decimal {
-      // little-endian twos-complement integer representation of complete value
-      // (ignoring precision) Always 16 bytes in length
-      bytes value = 1;
+      // the string representation.
+      string value = 1;
```

Comment:
Why not bytes?

Reply:
refer to the constructors of Decimal https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/types/Decimal.scala#L541-L564

I think we can use :
```
  def apply(unscaled: Long, precision: Int, scale: Int): Decimal =
    new Decimal().set(unscaled, precision, scale)

  def apply(value: String): Decimal = new Decimal().set(BigDecimal(value))
```

I just choose `string` for two reasons:
1,  it is convenient;
2, not sure about how to get the `unscaled` and `scale`  from python `decimal`